function output(error) {
	console.warn(error.stack);
}

process.on("unhandledRejection", (reason, line) => {
	output(new Error(line, reason));
});
process.on("warning", (warning) => {
	output(new Error(warning));
})
module.exports = function handle(client) {
	client.on("error", output);
}
